### PR TITLE
Setup openshift-metal3/dev-scripts for OpenShift CI

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: images/Dockerfile.ci [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM registry.svc.ci.openshift.org/origin/4.5:base
```
Comment (elfosardo) [inline images/Dockerfile.ci:1]: just as you're aware that this means it's going to run rhel7, are we ok with this ?

File: images/Dockerfile.ci [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+WORKDIR /dev-scripts
+COPY . .
+
+RUN yum update -y && \
+    yum install --setopt=tsflags=nodocs -y \
+    libvirt-libs openssl unzip jq openssh-clients && \
+    yum clean all && rm -rf /var/cache/yum/* && \
+    chmod g+w /etc/passwd
+
+RUN mkdir /output && chown 1000:1000 /output
```
Comment (elfosardo) [inline images/Dockerfile.ci:12]: add this to the precedent RUN

File: images/Dockerfile.ci (hunk 1 above)
Comment (stbenjam) [reply images/Dockerfile.ci:1]: I'm still thinking about how best to integrate this, but the base container OS probably doesn't matter here.

CI jobs need a container, but for e2e-metal-ipi, it runs on Packet anyway outside the OCP cluster.  The options for testing dev-scripts in CI are basically (1) have a container with dev-scripts content, and scp it over to the Packet host, (2) just have the CI job look at the PULL_* environment variables from CI and git clone on packet directly.

Comment (stbenjam) [reply images/Dockerfile.ci:1]: I think the CI folks do prefer option 1, since it's more like how everything else runs but I'm not sure it's worth it.

**Pull request merged**